src/hooks/useGetQuestion.ts

```diff
@@ -0,0 +1,23 @@
+import { useAppDispatch, useAppSelector } from "../state/reduxHooks";
+import axios from "axios";
+import { useEffect } from "react";
+import { API_URL } from "../settings";
+
+export function useCreateGame() {
+  const { code, questionStartCallTs } = useAppSelector(state => state.game);
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    if (!code) return;
+    if (questionStartCallTs === undefined) return; 
+    axios
+      .get(`${API_URL}/games/question/${code}`)
+      .then(response => {
+        console.log(response);
```

Comment:
We should dispatch the question name, choices, and correct answer index to the answerState from here. You'll need to add them to the question state, and verify what form they return using Postman and the console.log here. 

When you think you have everything done right, you can test this hook out by putting it inside the QuestionPage. We can temporarily put a button there which when pressed dispatches `createQuestion` (which we may want to rename `fetchQuestion`), thus triggering the hook. You can use the create game button as an example.

Reply:
Thanks Joel. I am currently dispatching the request data and I will verify what we get back once I use console.log and test the hook.

Reply:
Sounds good!